Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:alpine
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+RUN adduser -S sopel
+
+COPY requirements.txt /usr/src/app/
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . /usr/src/app
+
+USER sopel
+
+# OS X users will crash with permission errors if they use a
+#     volume over /home/sopel/.sopel - nothing we can do about it.
+# Workaround: Don't use OS X
+# https://github.com/boot2docker/boot2docker/issues/581
+# https://github.com/docker/kitematic/issues/351
+VOLUME [ "/home/sopel/.sopel" ]
```

Comment:
Because ~/.sopel does not exist when the `VOLUME` directive is run, docker created the directory owned by `root`. This leads to the following error on run: `ValueError: Invalid value for core.logdir: Value must be an existing or creatable directory.` May be best to just make it `VOLUME [ "/home/sopel" ]` since it should be empty (or super close to); or could explicitly `mkdir /home/sopel/.sopel` before the current `VOLUME` call (but after  `USER sopel`)